### PR TITLE
[Mobile] Create ios flavours

### PR DESCRIPTION
**Pull request by Mnoble-19**
#### Summary of Changes (What does this PR do?)
- Create iOS flavours i.e AirQo and AirQo Dev

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [x] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Checkout the branch
* Select the xcode Scheme section and choose among the two

#### What are the relevant tickets?
- [ticket_id](https://airqoteam.atlassian.net/browse/MOB-106)

#### Screenshots (optional)

### CODE REVIEW

**Comment by Baalmart** [comment]
Hi @Mnoble-19 , please note that this PR has some merge conflicts introduced by the most recent merges. Please act accordingly.

**Review by NoahNsimbe — APPROVED** [review]
LGTM. Thanks @Mnoble-19

File: mobile/ios/Runner/Info.plist [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>$(DEVELOPMENT_LANGUAGE)</string>
 	<key>CFBundleDisplayName</key>
-	<string>AirQo</string>
```
Comment (Baalmart) [inline mobile/ios/Runner/Info.plist:14]: What inspired this change to replacing a variable with an actual value? @Mnoble-19

Comment (Baalmart) [reply mobile/ios/Runner/Info.plist:14]: And where is AirQo_dev as mentioned in the PR description? @Mnoble-19

**Review by Baalmart — COMMENTED** [review]
Thanks @Mnoble-19 .

I have left a few comments....

I will be waiting for your response.....

File: mobile/ios/Runner/Info.plist (hunk 1 above)
Comment (Mnoble-19) [reply mobile/ios/Runner/Info.plist:14]: > What inspired this change to replacing a variable with an actual value? @Mnoble-19

Good morning @Baalmart , it's the other way round. This key holds the app name. Originally it was only one name for the app, already hard-coded. 
 As we're now using flavours, there is more than possible name  of the app i.e `AirQo` and `AirQo Dev`, thus I put variable and the name will be determined during the build

> And where is AirQo_dev as mentioned in the PR description? @Mnoble-19
I've added some screenshots to show the output

File: mobile/ios/Runner.xcodeproj/xcshareddata/xcschemes/airqodev.xcscheme [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Mnoble-19) [inline mobile/ios/Runner.xcodeproj/xcshareddata/xcschemes/airqodev.xcscheme:1]: @Baalmart , this is the file contains all the necessary information, concerning the airqodev scheme.

File: mobile/ios/Runner/Info.plist (hunk 1 above)
Comment (NoahNsimbe) [reply mobile/ios/Runner/Info.plist:14]: @Baalmart I think you comments have been responded to.

**Review by Baalmart — APPROVED** [review]
Thanks @Mnoble-19 , all looks good to me.

**Pull request merged**